Automattic/woocommerce-payments
# Introduced currency filter on deposits list page

Fixes #1340 

#### Changes proposed in this Pull Request

- Introduce currency filter on the deposits list page (shows up when more than one currency is available)

#### Testing instructions

* Connect the client to the server running locally or in the sandbox with server branch 710-gh-Automattic/woocommerce-payments-server
* Ensure the Stripe account has multiple settlement currencies (or change the currency of a deposit in DB to e.g. 'eur')
* Open deposits list page
** Ensure the filter is presented and changing the currency results in the proper deposits rendered
** Ensure sorting/pagination is preserved while changing the currency (for pagination you'll ned 25+ deposits in your account)
** Ensure the estimated deposits are counted properly in totals (make a test transaction for estimated deposit to appear)

-------------------
